src/Ryujinx.Ava/UI/ViewModels/TitleUpdateViewModel.cs

```diff
@@ -31,9 +30,8 @@ public class TitleUpdateViewModel : BaseModel
     {
         public TitleUpdateMetadata _titleUpdateWindowData;
         public readonly string     _titleUpdateJsonPath;
-        private VirtualFileSystem  _virtualFileSystem { get; }
-        private ulong              _titleId           { get; }
-        private string             _titleName         { get; }
+        private VirtualFileSystem VirtualFileSystem { get; }
+        private ulong TitleId { get; }
```

Comment:
```suggestion
        private VirtualFileSystem VirtualFileSystem { get; }
        private ulong TitleId { get; }
```

private > `_camelCase`

Reply:
Properties are not fields, so currently the default rule PascalCase applies: https://learn.microsoft.com/en-us/dotnet/fundamentals/code-analysis/style-rules/naming-rules#default-naming-styles

Should this still be changed?